CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## 0.10.0 - Unreleased
```

Comment:
This should be merged with the section below, since those changes haven't been released yet

Reply:
✔️